pysat/_instrument.py

```diff
@@ -91,10 +92,23 @@ class Instrument(object):
 
     Attributes
     ----------
-    bounds : (datetime/filename/None, datetime/filename/None)
-        bounds for loading data, supply array_like for a season with gaps.
-        Users may provide as a tuple or tuple of lists, but the attribute is
-        stored as a tuple of lists for consistency
+    platform
+    name
+    tag
+    inst_id
+    clean_level
+    pad
+    orbit_info
+    inst_module
+    temporary_file_list
+    strict_time_flag
+    directory_format
+    file_format
+    bounds : tuple
+        Tuple of two datetime objects or filenames indicating bounds for loading
```

Comment:
```suggestion
        Tuple of datetime objects or filenames indicating bounds for loading
```
You mention the case of bounds with gaps, which requires more than two total objects. I can see specifying two as it is the simplest case, could help users out. At the same time, specifying two when in general it isn't could be confusing for a different class of users. Raising this for consideration rather than a required change.